api/db/migrate/20250115214357_update_collaborator_role_values.rb

```diff
@@ -0,0 +1,14 @@
+class UpdateCollaboratorRoleValues < ActiveRecord::Migration[6.1]
+  def up
+    execute <<~SQL
+      UPDATE public.collaborators
+      SET role = 'author', priority = 0
+      WHERE role = 'creator';
+    SQL
+    execute <<~SQL
+      UPDATE public.collaborators
+      SET role = 'other', other_description = 'contributor', priority = 20
```

Comment:
With the priority updates, just making a note here that this migration will need to be updated.